src/test/metrics_flow_test.cpp

```diff
@@ -583,6 +710,66 @@ TEST_F(MetricFlowTest, ModelReady) {
     checkRequestsCounter(server.collect(), METRIC_NAME_REQUESTS_SUCCESS, dagName, 1, "REST", "ModelReady", "KServe", numberOfSuccessRequests);    // ran by real request
 }
 
+TEST_F(MetricFlowTest, RestV3Unary) {
+    HttpRestApiHandler handler(server, 0);
+    MockedServerRequestInterface stream;
+
+    EXPECT_CALL(stream, IsDisconnected())
+        .WillRepeatedly(::testing::Return(false));
+
+    for (int i = 0; i < numberOfAcceptedRequests; i++) {
+        std::string request = R"({"model": "dummy_gpt", "prompt": "Hello World"})";
+        std::string response;
+        HttpRequestComponents comps;
+        auto status = handler.processV3("/v3/completions", comps, response, request, &stream);
+        ASSERT_EQ(status, ovms::StatusCode::OK) << status.string();
+    }
+
+    // There is currently no possible scenario that V3 request get rejected.
```

Comment:
Is there plan for that to be possible? If not remove code below

Reply:
removed